baselines/FedPM/pyproject.toml

```diff
@@ -0,0 +1,139 @@
+[build-system]
+requires = ["poetry-core>=1.4.0"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "FedPM" # <----- Ensure it matches the name of your baseline directory containing all the source code
+version = "1.0.0"
+description = "Flower Baselines"
+license = "Apache-2.0"
+authors = ["The Flower Authors <hello@flower.dev>"]
```

Comment:
Feel free to update this with your contact info if you'd like. Also update the `description` with the name of your baseline or the title of the paper.